frontend/src/features/projects/spl/steps/SplStep.tsx

```diff
@@ -212,7 +212,9 @@ const SplStep = ({ formikRef }: IStepProps) => {
                       formikRef,
                       submitStatusCode,
                       getStatusTransitionWorkflow(submitStatusCode),
-                    );
+                    ).then((project: IProject) => {
+                      setValues(project);
```

Comment:
This is an example of the gotcha with this approach. If you have more then one way of updating the object used by the form (besides the formik submit) you won't get those changes automagically anymore.